modules/react-components/src/components/input/url-input.tsx

```diff
@@ -577,6 +587,17 @@ export const URLInput: FunctionComponent<URLInputPropsInterface> = (
                                         )
                                     }
                                 </Label>
+                                {
+                                    isUnsecureURL && (
+                                        <Popup
+                                            trigger={ <Icon name="exclamation triangle" color="yellow" /> }
```

Comment:
Shall we add a small margin left here. 

Try.
```suggestion
                                            trigger={ <Icon className="ml-1" name="exclamation triangle" color="yellow" /> }
```